.vscode/settings.json

```diff
@@ -18,4 +18,7 @@
     },
     "files.trimTrailingWhitespace": true,
     "C_Cpp.autoAddFileAssociations": false,
+    "cSpell.enableFiletypes": [
+        "!cpp"
+    ],
```

Comment:
Does this need to be added to Lime? It looks like something that belongs in user settings.

Reply:
It doesnt, looks like I used that extension while working on lime by accident. I'll remove this today (even right now if I can through GitHub...)